File: 1523-count-odd-numbers-in-an-interval-range/1523-count-odd-numbers-in-an-interval-range.js
/**
 * @param {number} low
 * @param {number} high
 * @return {number}
 */
var countOdds = function (low, high) {
    let count = 0;
    // for (let i = low; i <= high; i++) {
    //     if (i % 2 !== 0) {

    //     }
    // }
    while (low <= high){
        if (low % 2 == 0) {
            low++
        }
        else{
            count++
            low += 2
        }
    }
    return count

};

File: 1523-count-odd-numbers-in-an-interval-range/1523-count-odd-numbers-in-an-interval-range.cs
public class Solution {
    public int CountOdds(int low, int high) {
        int count = 0;
        while(low <= high){
            if(low % 2 != 0){
                count++;
                low +=2;
            }
            else{
                low++;
            }
        }
        return count;
        
    }
}